log/src/main/java/no/nav/common/log/JobRunner.java

```diff
@@ -0,0 +1,73 @@
+package no.nav.common.log;
```

Comment:
Burde denne ligge i en annen modul?

Reply:
Tenkte litt på det samme. Vi hadde en job-modul før, men har ikke lyst til å lage en ny modul kun for 1 klasse. Siden den ikke gjør noe annet enn logging, så tenkte jeg at `log` er like greit, men kanskje util-modulen hadde vært bedre?

Reply:
Da er jeg enig i at log-modulen bedre. En mulighet kunne vært å samle det med leader election, rename leader-election-modulen til job?

Reply:
Rename leader-election til job høres ut som en god ide